src/spring/azext_spring/app.py

```diff
@@ -447,11 +456,67 @@ def _log_application(cmd, client, no_wait, poller, resource_group, service, app_
                               since=300, timeout=10, get_app_log=_get_app_log_deploy_phase)
     except Exception:
         # ignore
-        return
+        pass
     if deployment_error:
         raise deployment_error
 
 
+def _print_deploy_process(client, poller, resource_group, service, app_name, deployment_name):
+    try:
+        deployment_resource = _get_deployment_ignore_exception(client, resource_group, service, app_name,
+                                                               deployment_name)
+        if deployment_resource is not None:
+            instance_count = deployment_resource.sku.capacity
+            rolling_number = max(1, instance_count // 4)
+            rounds = int(instance_count // rolling_number + 0 if instance_count % rolling_number == 0 else 1)
```

Comment:
Should we tell the round? Because if there are many instances under deployment, you cannot easy to tell which round we are in. E.g. we have 100 instance here. At the very beginning, you tell customer it will be finished in 4 round. Then customer will see 
"- 25 are creating, 75 are pending to be deleted"
"- 10 are created, 25 are creating, 65 are pending to be deleted"
....

There are more than 4 messages printed, that may confuse customer.

Reply:
for each round, only print one line message. it is done by if current_round != last_round: